SUMO-257738: Google Cloud Storage | Docs (#5271)
SumoLogic/sumologic-documentation: docs/integrations/google/cloud-storage.md

| fillmissing timeslice(1h)
```

### Sample metric queries

```
cloud.provider=gcp project_id=* metric=storage/object_count statistic=average 
| quantize using sum
| sum by project_id, bucket_name
```

## Collect logs for Google Cloud Storage

> This page describes the Sumo pipeline for ingesting logs from Google Cloud Platform (GCP) services and provides instructions for collecting logs from Google Cloud Storage.

Reviewer: ```suggestion
This section describes the Sumo Logic pipeline for ingesting logs from Google Cloud Platform (GCP) services, and explains how to collect logs from Google Cloud Storage.
```

> This page describes the Sumo pipeline for ingesting logs from Google Cloud Platform (GCP) services, and provides instructions for collecting logs from Google Cloud Storage.

Reviewer: ```suggestion
This section describes configuring the Sumo Logic pipeline for ingesting logs from Google Cloud Platform (GCP) services, and provides instructions for collecting logs from Google Cloud Storage.
```
